AsyncDisplayKit/Details/ASDataController.mm

```diff
@@ -200,25 +217,30 @@ - (void)_layoutNodesFromContexts:(NSArray<ASIndexedNodeContext *> *)contexts ofK
         dispatch_semaphore_signal(sema);
       });
       dispatch_semaphore_wait(sema, DISPATCH_TIME_FOREVER);
-      [self layoutLoadedNodes:subarray fromContexts:[contexts subarrayWithRange:NSMakeRange(j, batchCount)] ofKind:kind];
+      
+      [self _layoutNodes:subarray
+            fromContexts:contexts
+        inIndexesOfRange:NSMakeRange(0, subarray.count)
+                  ofKind:kind];
+      
     } else {
       allocationBlock();
       [_mainSerialQueue performBlockOnMainThread:^{
-        [self layoutLoadedNodes:subarray fromContexts:[contexts subarrayWithRange:NSMakeRange(j, batchCount)] ofKind:kind];
+        [self _layoutNodes:subarray
+              fromContexts:contexts
+          inIndexesOfRange:NSMakeRange(0, subarray.count)
+                    ofKind:kind];
```

Comment:
@appleguy What do you think creating also a helper method that does not have the index range as parameter that just calls through the one with index range as parameter with the whole nodes range as we here do?